src/test/java/com/epam/tc/hw3/page/IndexPage/IndexPage.java

```diff
@@ -0,0 +1,110 @@
+package com.epam.tc.hw3.page.IndexPage;
+
+import com.epam.tc.hw3.page.BasePage;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class IndexPage extends BasePage {
+    //DATA
+    private final List<String> TEXT_BENEFIT_LIST = Arrays.asList(
+            "To include good practices and ideas from successful EPAM project",
+            "To be flexible and customizable",
+            "To be multiplatform",
+            "Already have good base (about 20 internal and some external projects), wish to get more…");
+
+    //LOCATORS
+    @FindBy(className = "profile-photo")
+    private WebElement profilePhoto;
+
+    @FindBy(id = "name")
+    private WebElement nameLoginField;
+
+    @FindBy(id = "password")
+    private WebElement passLoginField;
+
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(id = "user-name")
+    private WebElement userName;
+
+    @FindBy(xpath = "//span[contains(@class,'icons-benefit')]")
+    private List<WebElement> imgIndexList;
+
+    @FindBy(className = "benefit-txt")
+    private List<WebElement> textBenefitList;
+
+    @FindBy(id = "frame")
+    private WebElement frameWthButton;
+
+    @FindBy(id = "frame-button")
+    private WebElement frameButton;
+
+    public IndexPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public void checkTitle() {
+        Assertions.assertThat(driver.getTitle()).isEqualTo("Home Page");
```

Comment:
It would be better if the assertion values were not hardcoded in the page object, try to parametrize the test

Reply:
Ok. I did.